docs/_static/nuts-auth.yaml

```diff
@@ -624,15 +565,12 @@ components:
         iss:
           type: string
           description: |
-            The issuer in the JWT is always the acting party, thus the care organization doing the request.
-            This is used to find the public key of the issuer from the Nuts registry.
+            The subject (not a Nuts subject) contains the urn of the custodian.
```

Comment:
I'd make 'URN' all caps, otherwise the subject could be a box with a porcelain object in it, which contains ashes of the deceased custodian after being burned.